packages/block-editor/src/store/private-actions.js

```diff
@@ -259,3 +259,16 @@ export function setBlockRemovalRules( rules = false ) {
 		rules,
 	};
 }
+
+/**
+ * Sets the client ID of the block settings menu that is currently open.
+ *
+ * @param {string} clientId The block client ID.
+ * @return {Object} Action object.
+ */
+export function setOpenedBlockSettingsMenu( clientId = '' ) {
```

Comment:
Do we really mean to have the default `clientId` as an empty string? I can see the action is set up to handle undefined `clientId` and the reducer will also return a `null`, and those are all three different values representing "empty", which seems like an inconsistent behavior. Should we make it consistent and ultimately only go with either of them?